### PR TITLE
Set up task list

### PR DESCRIPTION
**Pull request by s2jack**
Create blueprint of taskboard
Add Task template
Add Header
Add footer
Add heading
Add text input
Give basic styles

[Create modular Javascript Structure]

Create pageLoader.js to control page loading actions
Create taskClass module to contain constructor class of tasks
Create tasksUI.js to control tasks datsbase and loading methods

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,161 @@
...
+I would like to thank Microverse Family and all my coding partners.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
```
Comment (zeff96) [inline README.md]: - [x]  While you have made good effort to include the license, unfortunately am not able to open the license using the path provided. can adjust the path to the license. here is a suggestion on how you can solve this.
 ```suggestion
This project is [MIT](./LICENSE) licensed.
```

File: modules/tasksUI.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        // console.log('database has value');
+        const task = new Task(addTaskInput.value, dB.length);
+        const newDB = [...dB, task];
+        localStorage.setItem('tasksDB', JSON.stringify(newDB));
+        TasksUI.checkData();
+        TasksUI.loadTasksUI();
+      }
+    }
+  }
+}
```
Comment (zeff96) [inline modules/tasksUI.js]: - [x] Good work implementing this functionality :+1: , However, according to the projects [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md) for this milestone, you are required to set up a simple array of objects and iterate over it and render to the `DOM`
> It has an array of some simple to do tasks (array of objects) with each object containing three keys - description [string], completed [bool], index: [number]

> Example

`const tasks = [
  {
    description: 'An example description in string',
    completed: false,
    index: 10,
  },
];
`

**Review by zeff96 — CHANGES_REQUESTED** [review]
### Status: CHANGES REQUIRED ♻️ ♻️ ♻️ 
Hi @s2jack, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost

## Highlights
:heavy_check_mark: Linters passing
:heavy_check_mark: Correct usage of Github flow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/tasksUI.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+        // console.log('database has value');
+        const task = new Task(addTaskInput.value, dB.length);
+        const newDB = [...dB, task];
+        localStorage.setItem('tasksDB', JSON.stringify(newDB));
+        TasksUI.checkData();
+        TasksUI.loadTasksUI();
+      }
+    }
+  } */
+}
```
Comment (roshan-bajgain) [inline modules/tasksUI.js:85]: - [ ] [optional] Please remove the commented line of code as it make the code dirty.

**Review by roshan-bajgain — APPROVED** [review]
Hi @s2jack, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
- Good job on implementing the changes ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by s2jack** [comment]
Create blueprint of taskboard
Add Task template
Add Header
Add footer
Add heading
Add text input
Give basic styles
Create pageLoader.js to control page loading actions
Create taskClass module to contain constructor class of tasks
Create tasksUI.js to control tasks datsbase and loading methods
[Add tasks object Array](https://github.com/s2jack/To-Do-List/commit/db8e6c15a295618330a4950561a9dfee92c56374) 

Fix README licence issue
[Add tasks object Array](https://github.com/s2jack/To-Do-List/commit/db8e6c15a295618330a4950561a9dfee92c56374) 

Fix README licence issue

**Pull request merged**